packages/babel-plugin-glaze/src/index.ts

```diff
@@ -0,0 +1,305 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable no-param-reassign */
```

Comment:
I had to turn off some of the eslint lint for this file. Was a bit lazy to properly handle them and actually I'm not sure if that makes sense to enable them for this file...

Reply:
I'll check whether enabling them is worth it. I prefer enabling/disabling rules on a per-line or per-project basis. The latter can be done through `.eslintrc`.

Reply:
Also, thank you for highlighting this! 😊